.github/workflows/tools.yml

```diff
@@ -78,6 +78,16 @@ jobs:
                 echo "NEW_VERSION=$NEW_VERSION" >> $GITHUB_ENV
                 ./tools/update-undici.sh
               fi
+          - id: postject
+            subsystem: deps
+            label: dependencies
+            run: |
+              NEW_VERSION=$(npm view postject dist-tags.latest)
+              CURRENT_VERSION=$(node -p "require('./deps/postject/src/package.json').version")
+              if [ "$NEW_VERSION" != "$CURRENT_VERSION" ]; then
+                echo "NEW_VERSION=$NEW_VERSION" >> $GITHUB_ENV
+                ./tools/update-undici.sh
```

Comment:
```suggestion
                ./tools/update-postject.sh
```

Reply:
Thanks, incorporated this in https://github.com/nodejs/node/pull/45066